tests/testthat/test_clean_report_sources.R

```diff
@@ -1,8 +1,7 @@
 
+context("Test removal of undesirable files and folders present report_sources/")
```

Comment:
Shorter context (usually the function name) is preferable here

Reply:
✅